### ISSUE TITLE
Coloring

### ISSUE THREAD

**Issue by Zatheyll** [nyaadevs/nyaa#28]
This is kind of a stupid one, but I and many others love using the dark theme, but the trusted torrents color looks terrible with the dark theme, maybe turn it to a darker green?

**Comment by Nazonaze** [comment]
The dark theme (and the layout in general) is very much work-in-progress, and stuff like this will be improved on.

**Comment by HitmanJ** [comment]
Actually, those colors should be darker in both themes.

**Comment by spillerrec** [comment]
My Suggestion:
![nyaa-dark-edit](https://cloud.githubusercontent.com/assets/2292170/26033815/2a9cfabe-38b3-11e7-95de-ec8f2e445e3e.png)
Colors are just changed to have a HSL lightness of 20%, i.e. the same as the default background color. Link color was changed to `hsla(207, 102%, 65%, 1)` to increase contrast a bit, but that could probably be tweaked a bit more.

`success`/`danger` colors are applied on the `tr` only, and then I changed odd-row rule so it can be applied on all colors:
```css
.table-striped>tbody>tr:nth-of-type(odd) > *
   { background-color: rgba(255,255,255,0.08); }
```

The message at the top should probably also get a dark variant.

**Comment by HitmanJ** [comment]
Instead of red, remakes should be orange so that people won't think those releases are bad, unapproved, or whatever.

**Comment by DarkDullahan** [comment]
I agree with HitmanJ. I feel to keep confusions to a minimum the color coding should match Nyaa.se as closely as possible, as both themes would have no issues with that color coding style.

**Comment by spillerrec** [comment]
I don't know what I'm doing, but here is an update:
![nyaa-dark-edit-v3](https://cloud.githubusercontent.com/assets/2292170/26080804/70755c6e-39c8-11e7-88b7-2a15172bf5d7.png)
I'm sure the remake/trusted colors are horrible for color blind people. Seed count has to be brighter to work out properly, but this is specified using element styles instead of CSS, making it hard to change dynamically.
Anyway, if the CSS wasn't minified we could actually make incremental pull-requests with diffs that are meaningful. I'm at least not seeing the source of the minified CSS files.

**Comment by RUSshy** [comment]
dark mode font colors are illisible, make it white-smoke, gray or yellowish, not glowing blue, and not white

**Comment by RUSshy** [comment]
please learn best practice for dark mode, and why people want dark mode, not for it being sexy, it is for it being readable at night *READABLE*

**Comment by TheAMM** [comment]
@RUSshy Calm down, the current dark mode is the bootstrap default which clashes with our row colors. Nobody has bothered to work on it yet - it's not due to not knowing what a dark theme is meant for.

**Comment by Arylide** [comment]
Everyone involved sucks at design stuff, PRs welcome for this.

**Comment by CounterPillow** [comment]
There is one in https://github.com/nyaadevs/nyaa/pull/184 but I'm not sure what the status of it is

**Comment by Azureum** [comment]
The new dark colors look very bad. Especially the green color. I suggest using #2e3e3e for green shade and #3e2e2e for red shade. This way they will blend well with other rows.
Example:
![sketch](https://user-images.githubusercontent.com/20842882/30031306-73b4fb30-91ae-11e7-97df-0d2814ac3d9e.png)
Compared to the present:
![sketch2](https://user-images.githubusercontent.com/20842882/30031323-84ca858e-91ae-11e7-9b9d-6e3375d1c063.png)

**Comment by CounterPillow** [comment]
@Azureum that looks almost too subtle for my taste.

**Comment by nijynot** [comment]
<img width="1218" alt="6bd30b6c6c266a142ef8d9eede80adc5 1" src="https://user-images.githubusercontent.com/16340977/30512873-037639d4-9af9-11e7-8b8a-1658a6f664ed.png">
<img width="1218" alt="c01d708fe5fc2a3bee91b39751061a0c 1" src="https://user-images.githubusercontent.com/16340977/30512874-03888670-9af9-11e7-819b-d90c0892b820.png">
Removed styling of odd rows as it made the ui seem more cluttered, especially when success- and danger rows were in the table. As the magnet icon is now colored, it won't be hard too see which row is which. Also made the blue in the search button more vibrant and fixed the contrast issue in #365.

**Comment by S-ed** [comment]
I'm using this kind of coloring:
![nyaa-fullpage](https://user-images.githubusercontent.com/3008353/30915445-8bd06d8c-a39f-11e7-9281-7a26679430f6.png)
But as was mentioned in #365 , there may be an issue for low contrast (my theme has low contrast, especially for visited links).
Also, as I mentioned in #369 there're no sources for dark bootstrap CSS (config.json).

**Comment by Inverness** [comment]
My issue with the dark theme is that when loading a page it is initially white before the dark theme is applied again. This is very irritating on the eyes to have the screen flash white each time you navigate to another page.

**Comment by S-ed** [comment]
@Inverness Have you switched default theme to dark (switch on the bottom of the page)? Or it's user one that flicking?

**Comment by Inverness** [comment]
@S-ed I used the switch at the bottom of the page.

**Comment by S-ed** [comment]
@Inverness 
Yep. Seems it happens on Chromium based browsers (Opera/Chrome).
You could make new issue to report it.

**Comment by CounterPillow** [comment]
white flashes aren't really something that can be fixed unless the stylesheet selection is done server-side, which is too much complexity for something this insignificant imho
